packages/app/src/app/pages/Dashboard/Header/index.tsx

```diff
@@ -50,10 +50,8 @@ export const Header: React.FC<HeaderProps> = React.memo(
           boxSizing: 'border-box',
           fontFamily: 'Inter, sans-serif',
           height: 12,
-          backgroundColor: 'titleBar.activeBackground',
+          backgroundColor: 'transparent',
```

Comment:
Can be removed, default has no `backgroundColor` I think.

Reply:
✅